roles/openshift_examples/files/examples/v1.2/xpaas-streams/jboss-image-streams.json

```diff
@@ -152,8 +152,8 @@
                             "version": "1.3"
                         },
                         "from": {
-                          "Kind": "DockerImage",
-                          "Name": "registry.access.redhat.com/jboss-eap-7-beta/eap70-openshift:1.3"
+                          "kind": "DockerImage",
+                          "name": "registry.access.redhat.com/jboss-eap-7-beta/eap70-openshift:1.3"
                         }
```

Comment:
@sdodson, I think the reason these passed testing is because they ran against 3.0 and 3.1.  It looks like for 3.2 it breaks validation.